conda/recipes/libcudf/recipe.yaml

```diff
@@ -0,0 +1,233 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split('.'))[:2] | join('.') }}
+  cuda_version: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[:2] | join(".") }}
+  cuda_major: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+recipe:
+  name: libcudf-split
+
+cache:
+  source:
+    path: ../../..
+
+  requirements:
+    build:
+      - ${{ compiler('c') }}
+      - ${{ compiler('cxx') }}
+      - ${{ compiler('cuda') }} =${{ cuda_version }}
+      - cuda-version =${{ cuda_version }}
+      - cmake ${{ cmake_version }}
+      - ninja
+      - ${{ stdlib("c") }}
+    host:
+      - librmm =${{ minor_version }}
+      - libkvikio =${{ minor_version }}
+      - if: cuda_major == "11"
+        then:
+          - cudatoolkit
+          - libcufile ${{ cuda11_libcublas_host_version }}
+          - libcufile-dev ${{ cuda11_libcublas_host_version }}
+          - libcurand ${{ cuda11_libcurand_host_version }}
+          - libcurand-dev ${{ cuda11_libcurand_host_version }}
+          - cuda-nvrtc =${{ cuda_version }}
+          - cuda-nvrtc-dev =${{ cuda_version }}
+          - cuda-nvtx =${{ cuda_version }}
+        else:
+          - cuda-nvrtc-dev
+          - cuda-nvtx-dev
+          - libcurand-dev
+          - if: linux64
+            then:
+              - libcufile-dev
+      - cuda-version =${{ cuda_version }}
+      - nvcomp ${{ nvcomp_version }}
+      - dlpack ${{ dlpack_version }}
+      - librdkafka ${{ librdkafka_version }}
+      - flatbuffers =${{ flatbuffers_version }}
+      - rapids-logger =0.1
+      - zlib ${{ zlib_version }}
+
+  build:
```

Comment:
It feels odd to me that this section is below the requirements. Is that how rattler examples are structured? It's normally `build` before `requirements` in most conda recipes.

Reply:
swapped